camayoc/tests/qpc/ui/conftest.py

```diff
@@ -29,24 +32,34 @@ def pytest_collection_modifyitems(config, items):
 def browser(request):
     """Selenium instance.
 
-    The current configuration takes advantage of a remote
-    webdriver for configurable setup. As of writing, this supports
-    a standalone chrome container. The following command can be used
-    to spin up the container (one line, no line breaks)
-
-    'docker run -d -p 4444:4444 -v /dev/shm:/dev/shm
-    selenium/standalone-chrome:3.13.0-argon'
+    See README for configuration of remote browser containers.
     """
-    chrome_options = webdriver.ChromeOptions()
-    chrome_options.add_argument('--headless')
-    chrome_options.add_argument('--disable-gpu')
-    chrome_options.add_argument('--no-sandbox')
-    chrome_options.add_argument('--allow-insecure-localhost')
-    driver = webdriver.Remote(
-        'http://127.0.0.1:4444/wd/hub',
-        desired_capabilities=chrome_options.to_capabilities())
+    # Default to chrome for UI testing.
+    try:
+        os.environ['SELENIUM_DRIVER']
+    except KeyError:
+        os.environ['SELENIUM_DRIVER'] = 'Chrome'
+
+    driver_type = os.environ['SELENIUM_DRIVER']
+    if driver_type == 'Chrome':
+        chrome_options = webdriver.ChromeOptions()
+        chrome_options.add_argument('--headless')
```

Comment:
maybe you can have an environment variable: `DEBUG_UI` to decide whether or not to go headless?

Reply:
```
if not os.environ.get('DEBUG_UI', False):
    chrome_options.add_argument('--headless')
```